Rakefile

```diff
@@ -8,21 +8,23 @@ rescue LoadError
 end
 
 require 'solr_wrapper'
+require 'solr_wrapper/rake_task'
 require 'engine_cart/rake_task'
 require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
+require 'solr_wrapper/rake_task'
```

Comment:
Double require?

Reply:
Good catch. I'll remove that line in a follow on PR.